packages/pwa-kit-extension-sdk/src/configs/webpack/application-extensions-loader.ts

```diff
@@ -37,8 +37,12 @@ export default function ApplicationExtensibilityLoader(
     // Get the installed and configured application extensions as well as the requested
     // target type. For web targets the loader takes advantage of react-loadabled but node
     // targets (server) do not require this optimization.
-    const data = this.getOptions()
+    const data = {
+        ...this.getOptions(),
+        installed: getPackageNamesOfInstalledExtensions()
```

Comment:
🤔 Adding the installed package names here, inline, in the loader code itself and not passed in will most likely make testing more difficult as you may have to mock the config object. That is why using dependency injection, aka passing in the information, usually leads to a more flexible and easier to test API.